build.gradle

```
@@ -184,6 +184,13 @@ task generateTypes {
     dependsOn generateApiConstants
 }
 
+task execScript(type: JavaExec) {
+    description "Run user defined scripts specified by project variable - userScript"
+
+    classpath = sourceSets.test.runtimeClasspath
+    mainClass = project.hasProperty("userScript") ? "teammates.client.scripts.${project.property('userScript')}" : "teammates.client.scripts.PlaceholderUserScript"
```

Just wondering, would this work instead?

```suggestion
    if (project.hasProperty("userScript")) {
        mainClass = "teammates.client.scripts.${project.property('userScript')}"
    }
```

---

This seems to work as expected. I don't think there is a need for a "Placeholder Script".

---

This works as well, didn't know that mainClass does not have to be specified! Made changes accordingly